.storybook/preview-head.html

```diff
@@ -1,18 +1,53 @@
 <style>
-  @font-face {
-    font-family: 'IBM Plex Mono';
-    font-style: normal;
-    font-weight: 400;
-    src: url('static/fonts/IBMPlexMono-Regular.woff') format('woff'),
-    url('static/fonts/IBMPlexMono-Regular.woff2') format('woff2'),
-    url('static/fonts/IBMPlexMono-Regular.ttf') format('truetype');
-  }
-  @font-face {
-    font-family: 'IBM Plex Sans';
-    font-style: normal;
-    font-weight: 400;
-    src: url('static/fonts/IBMPlexSans-Regular.woff') format('woff'),
-    url('static/fonts/IBMPlexSans-Regular.woff2') format('woff2'),
-    url('static/fonts/IBMPlexSans-Regular.ttf') format('truetype');
-  }
-</style>
\ No newline at end of file
+    @font-face {
+        font-family: 'IBM Plex Mono';
+        font-style: normal;
+        font-weight: 400;
+        src: url('static/fonts/IBMPlexMono-Regular.woff') format('woff'),
+            url('static/fonts/IBMPlexMono-Regular.woff2') format('woff2'),
+            url('static/fonts/IBMPlexMono-Regular.ttf') format('truetype');
+    }
+    @font-face {
+        font-family: 'IBM Plex Sans';
+        font-style: normal;
+        font-weight: 400;
+        src: url('static/fonts/IBMPlexSans-Regular.woff') format('woff'),
+            url('static/fonts/IBMPlexSans-Regular.woff2') format('woff2'),
+            url('static/fonts/IBMPlexSans-Regular.ttf') format('truetype');
+    }
+    body {
+        font-size: 16px;
+    }
+    .sbdocs .sbdocs-h1,
+    .sbdocs .sbdocs-h2,
+    .sbdocs .sbdocs-h3,
+    .sbdocs .sbdocs-h4,
+    .sbdocs .sbdocs-h5,
+    .sbdocs .sbdocs-h6,
+    .sbdocs .sbdocs-p,
+    .sbdocs .sbdocs-ol,
+    .sbdocs .sbdocs-ul {
+        font-family: 'IBM Plex Sans';
+        line-height: 1.5;
+    }
+    .sbdocs .sbdocs-h1 {
+        margin-top: 0;
+        font-weight: 700;
+        font-size: 2.488rem;
+    }
+    .sbdocs .sbdocs-h2 {
+        font-size: 2.074rem;
+    }
+    .sbdocs .sbdocs-h3 {
+        font-size: 1.728rem;
+    }
+    .sbdocs .sbdocs-h4 {
+        font-size: 1.44rem;
+    }
+    .sbdocs .sbdocs-h5 {
+        font-size: 1.2rem;
+    }
+    .sbdocs .sbdocs-p {
+        margin-bottom: 1rem;
+    }
```

Comment:
O interesting so you are setting globals here.

Reply:
It's kind of inelegant, but it's the "easiest" way to access the defaults for Storybook documentation. The Markdown only files (not wrapped in a `<Story>` element) only get the styles from this preview, there's not a way to to override them otherwise. 

It's not my favorite thing, but it's a minor tradeoff (hopefully).